about.html

```diff
@@ -0,0 +1,174 @@
+                                <div class="history-card-content">
+                                    <h3 class="history-card-title">2022</h3>
+                                    <p class="history-card-p">
+                                        Increase your (international) visibility.
+                                    </p>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="history-card card-2">
+                            <div class="card-overlay">
+                                <div class="history-card-content">
+                                    <h3 class="history-card-title">2021</h3>
+                                    <p class="history-card-p">
+                                        Increase your (international) visibility.
+                                    </p>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <!-- Partner section -->
+        <section class="partner">
+            <div class="container">
+                <div class="partner-content">
+                    <h2 class="section-title partner-title">Partner</h2>
+                    <div class="partner-list">
+                        <div class="partner-item">
+                            <div class="partner-item-img">
+                                <img src="img/partner-1.png" alt="">
+                            </div>
+                        </div>
+                        <div class="partner-item">
+                            <div class="partner-item-img">
+                                <img src="img/partner-2.png" alt="">
+                            </div>
+                        </div>
+                        <div class="partner-item">
+                            <div class="partner-item-img">
+                                <img src="img/partner-3.png" alt="">
+                            </div>
+                        </div>
+                        <div class="partner-item">
+                            <div class="partner-item-img">
+                                <img src="img/partner-4.png" alt="">
+                            </div>
+                        </div>
+                    </div>
+                    <div class="partner-overlay"></div>
+                </div>
+            </div>
+        </section>
+    </div>
+    <footer class="footer">
+        <div class="container">
+            <div class="footer-content">
+                <div class="footer-logo">
+                    <h1 class="logo">event<span>Work</span></h1>
+                </div>
+                <div class="footer-adress">
+                    <p>2015 Creative Commons Korea <br> Some Rights Reserved</p>
+                </div>
+            </div>
+        </div>
+    </footer>
```

Comment:
- [ ] It will be better if the footer follows the guideline in the provided design
   - Desktop Size
             About Page: It has a dark background.
             Home Page: It has a white background as you did.